### ISSUE TITLE
💅 False positive for Bun's `util` import at `lint/style/useNodejsImportProtocol`

### ISSUE THREAD

**Issue by phenomen** [biomejs/biome#2389]
### Environment information

```bash
CLI:
  Version:                      1.6.4
  Color support:                true

Platform:
  CPU Architecture:             x86_64
  OS:                           windows

Environment:
  BIOME_LOG_DIR:                unset
  NO_COLOR:                     unset
  TERM:                         "xterm-256color"
  JS_RUNTIME_VERSION:           "v20.11.1"
  JS_RUNTIME_NAME:              "node" <-- actually it should be "bun"
  NODE_PACKAGE_MANAGER:         unset

Biome Configuration:
  Status:                       Loaded successfully
  Formatter disabled:           false
  Linter disabled:              false
  Organize imports disabled:    false
  VCS disabled:                 false

Linter:
  Recommended:                  true
  All:                          false
  Rules:
```

### Rule name

lint/style/useNodejsImportProtocol

### Playground link

https://biomejs.dev/playground/?code=LwAvACAAaAB0AHQAcABzADoALwAvAGIAdQBuAC4AcwBoAC8AZwB1AGkAZABlAHMALwBwAHIAbwBjAGUAcwBzAC8AYQByAGcAdgAKAGkAbQBwAG8AcgB0ACAAewAgAHAAYQByAHMAZQBBAHIAZwBzACAAfQAgAGYAcgBvAG0AIAAiAHUAdABpAGwAIgA7AA%3D%3D

### Expected result

Leave "util" import as-is.

### Code of Conduct

- [X] I agree to follow Biome's Code of Conduct

**Comment by Conaclos** [comment]
Have you a reference to Bun docs for this module? Is this a valid bun import? Is `bun:util` valid?

**Comment by phenomen** [comment]
Nevermind, `node:util` works with Bun (I guess it's polyfilled), so it doesn't matter which import to use.

**Comment by Conaclos** [comment]
I think that `util` and `node:util` are the same. Bun supports both bare node builtins and `node:` prefixed builktins.
